tests/unit/src/test/scala/tests/ScalaToplevelSuite.scala

```diff
@@ -37,17 +37,18 @@ class ScalaToplevelSuite extends BaseSuite {
        |  def foo: Int
        |  class Z
        |
-       |class B(val v: String):
+       |class B(val v: String, g: String):
        |  trait X
        |  def foo: Int
        |
-       |trait C
+       |trait C(p: String, val i: Int)
```

Comment:
Would it be a good idea to introduce a test for the enum as well (based on the comment in https://github.com/scalameta/metals/pull/5358#discussion_r1237170784)? :slightly_smiling_face:

Reply:
Turns out it's more than a good idea since it doesn't work. Also @tgodzik, @jkciesluk shouldn't we emit cases in enums? I'm not sure why we don't.

Reply:
I think we should, but I don't think it was ever handled. Should be possible to do but we should make sure normal case statements aren't added by msitake.